### PR TITLE
portalicious: implement basic auth strategy

### PR DESCRIPTION
**Pull request by aberonni**
[AB#31186](https://dev.azure.com/redcrossnl/7d17f2a9-3f18-40f1-8c29-98568fbca7e4/_workitems/edit/31186)

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added tests wherever relevant
- [x] I have made sure that all automated checks pass before requesting a review
- [x] I do not need any deviation from our PR guidelines

### CODE REVIEW

File: interfaces/Portalicious/src/app/pages/change-password/change-password.page.html [hunk 1]
```diff
@@ -3,49 +3,6 @@
   i18n-pageTitle="@@page-title-change-password"
 >
   <p-card styleClass="max-w-xl">
-    <app-form-default
```
Comment (elwinschmitz) [inline interfaces/Portalicious/src/app/pages/change-password/change-password.page.html:6]: Where is the code for the `ChangePasswordComponent`?
Did you forget to commit it, or is it already somewhere else in the codebase?

File: interfaces/Portalicious/src/app/pages/change-password/change-password.page.html [hunk 2]
```diff
@@ -3,49 +3,6 @@
...
-        [errorMessage]="formFieldErrors()('confirmPassword')"
-      >
-        <p-password
-          formControlName="confirmPassword"
-          [toggleMask]="true"
-          [feedback]="false"
-        />
-      </app-form-field-wrapper>
-    </app-form-default>
+    <ng-container *ngComponentOutlet="ChangePasswordComponent"></ng-container>
```
Comment (elwinschmitz) [inline interfaces/Portalicious/src/app/pages/change-password/change-password.page.html:6]: This way, maybe some day, we could return this page back to its "User settings"-purpose.. ;)

**Review by elwinschmitz — APPROVED** [review]

File: interfaces/Portalicious/src/app/services/auth/strategies/basic-auth/basic-auth.login.component.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      [feedback]="false"
+    />
+  </app-form-field-wrapper>
+</app-form-default>
+<p class="mt-4">
+  <span i18n>Forgot your password?</span>&nbsp;
+
+  <a
+    i18n
+    href="mailto:support@510.global?subject=Request Password reset for 121 Platform"
```
Comment (elwinschmitz) [inline interfaces/Portalicious/src/app/services/auth/strategies/basic-auth/basic-auth.login.component.html:42]: Sanity-check with @tijsziere : Don't we want to use `support@121.global` everywhere?

Comment (aberonni) [reply interfaces/Portalicious/src/app/services/auth/strategies/basic-auth/basic-auth.login.component.html:42]: I followed the design, but we can add it back to the main page when we implement SSO if we have to.

Comment (aberonni) [reply interfaces/Portalicious/src/app/services/auth/strategies/basic-auth/basic-auth.login.component.html:42]: Ah sorry, I misunderstood the original comment. I will fix this in another PR.

**Pull request merged**